packages/rpc-core/src/response-patcher-allowed-numeric-values.ts

```diff
@@ -22,6 +22,7 @@ export function getAllowedNumericKeypathsForNotification(): AllowedNumericKeypat
     if (!memoizedNotificationKeypaths) {
         memoizedNotificationKeypaths = {
             slotsUpdatesNotifications: [['timestamp']],
+            voteNotifications: [['timestamp']],
```

Comment:
Typed as `i64` on the server, so I don't think you should cast it, unless I'm missing something?

Reply:
I guess I got confused since the `UnixTimestamp` is a `number` not `bigint`, so I thought we always have to cast it

Reply:
I'm also confused why these numbers are typed the way they are. Casting to number _seems_ reasonable!